src/Client/RedisClientConfig.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ChunkingRedisClient.Client
+{
+    public sealed class RedisClientConfig
```

Comment:
I know what you mean about this, I just prefer interfaces since it leaves it totally up to the end user to define how they want to compose/setup the retrieval of config settings

Reply:
I can see the charm of interfaces but even with the class (atleast in its current form), the developer is still in full control of configuring the connection strings and since redis connection strings are literally just strings, I'd suggest deferring introduction of abstractions. Introducing an interface at a later stage can be done in a non-breaking way.

Ofcourse the alternative is to complicate matters further and converting the `string` property to a `Func<string>` property but even that doesn't require an interface :). Let me know what you think, and if you'd still rather have i'faces then I will change it, but I still think code should be kept as simple as possible.

Reply:
That's fine, let's go with the class version